src/chrome/content/rules/Bitly_branded_short_domains.xml

```diff
@@ -10967,7 +10967,6 @@ This file generated by Bitly on Fri, 10 Nov 2017 17:24:38 UTC
 	<target host="go.bcoach.info" />
 	<target host="go.beardben.com" />
 	<target host="go.beckatl.com" />
-	<target host="go.beeming.net" />
```

Comment:
This target is not yet preloaded.

Reply:
Hum, that's weird, I use `hsts-prune` to remove those targets. I didn't remove them myself

Reply:
Where is it missing?

Reply:
https://dxr.mozilla.org/mozilla-central/source/security/manager/ssl/nsSTSPreloadList.inc

Reply:
Ah, got it!

Reply:
As far as I can tell, they only list domains. They don't enumerate all subdomains. Otherwise, there would be much entries starting with `www`.